opendistro-elasticsearch-security.release-notes

```diff
@@ -1,4 +1,9 @@
-## 2020-03-05, Version 0.9.0.6 (Current)
+## 2020-03-31, Version 0.9.0.7 (Current)
+
+- Fixed bug which caused user to lose roles on account password update
+- Performance improvement by implementing faster version of implies type perm
```

Comment:
I can't see the performance improvement as part of this PR

Reply:
It is present in the branch but a release was not made for it hence updating the release notes now.

Reply:
Please update the list of fixes.